midi/midi/music_21.py

```diff
@@ -110,34 +107,52 @@ def get_raw_midi_features(events, tempos):
     return features
 
 
-def check_number_of_tracks(f, filename):
-    if len(f) != NO_TRACKS:
+def check_number_of_tracks(f, filename, number):
+    if len(f) != number:
         raise ValueError('number of tracks ({}) in {} not matching the expected value ({})'
-                         .format(len(f), filename, NO_TRACKS))
+                         .format(len(f), filename, number))
 
 
-def get_features(filepath, filename):  # TODO: add a mode switcher
-    # file_input = np.load(filepath, allow_pickle=True)
+def preprocess_features(filepath, filename, check_tracks=False, numer_of_tracks=0):
+    # file_input = np.load(filepath, allow_pickle=True)  # TODO: choose input method
     # file_input = file_input.tolist()
     file_input = get_sequence_of_notes(filepath, Mode.BOOLEANS, False, True)
 
-    check_number_of_tracks(file_input, filename)
+    if check_tracks:
+        check_number_of_tracks(file_input, filename, numer_of_tracks)
     tempo_array = export_tempo_array(filepath)
     event_lists = get_event_lists(file_input)
 
-    # feature_list = get_tonal_midi_features(event_lists, tempo_array)
+    return event_lists, tempo_array
+
+
+def get_midi_features(filepath, filename, check_tracks=False, numer_of_tracks=0):
```

Comment:
typo `numer` -> `number`

Reply:
issue resolved in last commit (b511e4a4f5fde8eefa268bfec2d8c57854f2ec80)